Mechatronics.tex

```diff
@@ -1969,13 +1967,13 @@ \section{Tolerance Stacking}
 
 \section{Loosening}
 
-\textit{Positive retention} is ensuring that vibrations and loads will not effect the positioning of components. This means avoiding slots for adjustment (that do not have additional locking schemes like cams), and bolts that must be ``just properly tightened" in order to work properly.
+\par{\textit{Positive retention} is ensuring that vibrations and loads will not effect the positioning of components. This means avoiding slots for adjustment (that do not have additional locking schemes like cams), and bolts that must be ``just properly tightened" in order to work properly.
```

Comment:
Why \par{ ?
I have opinions about leading paragraph indents. They can and should be done globally rather than using \par